authors/apps/comments/views.py

```diff
@@ -33,6 +33,23 @@ def post(self, request, **kwargs):
         article = self.get_article(kwargs['article_id'])
         comment = request.data.get('comment', {})
         comment['article'] = article.id
+        first_index = comment.get('first_index', 0)
+        last_index = comment.get('last_index', 0)
+        if "first_index" in comment and "last_index" in comment:
+            if not isinstance(first_index, int) or not isinstance(last_index, int):
```

Comment:
@engjames Can this validation to check if these values provided(`validate_first_index, validate_last_index`) are ints be placed in the serialiser class.